src/main/java/com/agilethought/atsceapi/controller/UserController.java

```diff
@@ -1,20 +1,33 @@
-package com.agilethought.atsceapi.controller;
-
-import com.agilethought.atsceapi.model.LoginData;
-import com.agilethought.atsceapi.model.User;
-import com.agilethought.atsceapi.service.UserService;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.web.bind.annotation.*;

```

Comment:
these imports are including in this one. Maybe it is better use `import org.springframework.web.bind.annotation.*;`  instead of 
`import org.springframework.web.bind.annotation.PostMapping;
import org.springframework.web.bind.annotation.RequestBody;
import org.springframework.web.bind.annotation.ResponseStatus;
import org.springframework.web.bind.annotation.RestController;`